bot/engine/src/test/kotlin/engine/config/ConfiguredStoryHandlerTest.kt

```diff
@@ -51,34 +51,6 @@ class ConfiguredStoryHandlerTest {
         override val connectorType: ConnectorType = ConnectorType.none
     ) : ConnectorMessage
 
-    @Test
```

Comment:
please reuse this test :)

Reply:
Story-rules redirection is now tested by `BotDefinitionWrapperTest`.
(`BotBusTest` still tests the logic with `SWITCH_STORY_BUS_KEY`.)